Unity-Technologies/ScriptableRenderPipeline
# Fix for Fogbugz case 1107087. Reimporting shadergraphs

### Purpose of this PR
Fix for fogbugz case 1107087. Reimporting shadergraphs
https://fogbugz.unity3d.com/f/cases/1107087/

---
### Release Notes
This should fix the re-importing of shaders happening when using LWRP

---
### Testing status
Katana
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=sg%2Ffix-constantly-recompiling&automation-tools_branch=add-platform-filter&unity_branch=trunk

**Manual Tests**: What did you do?
Update Master to latest and Trunk to latest. The issue was still clearly there. Any changes re-imported the shadergraphs. 
Added the fix and Voila it is no longer re-importing every change.

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: Low

**Halo Effect**: Low

---
### Comments to reviewers
Notes for the reviewers you have assigned.
